index.html

```diff
@@ -0,0 +1,128 @@
+                  </li>
+                  <li class="nav-item">
+                    <a href="#" class="campaign">CC campaign</a>
+                  </li>
+                </ul>
+            </nav>
+        </header>
+        <div class="headline">
+            <h3 class="hello">"Hello! Sharing World!"</h3>
+            <h1 class="title">The Event Listener Music Festival 2015</h1>
+            <div class="intro-box">
+                <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+            </div>
+            <h2 class="dates">2015.10.15(THU)~16(FRI)</h2>
+            <p>@Gwangalli Beach, Busan</p>
+        </div>
+    </section>
+    <section id="main-program">
+        <h3 class="including">Including</h3>
+        <div class="include-container">
+        <ul class="include-list">
+            <li><i class="fa-sharp fa-solid fa-music"></i></li>
+            <li class="offer">Music</li>
+            <li>We made history by discovering new musical promises and presenting the most renowned names in music worldwide. With a careful and coherent lineup, come join us in this adventure and share the message of music.</li>
+        </ul>
+        <ul class="include-list">
+            <li><i class="fa-solid fa-campground"></i></li>
+            <li class="offer">Camping</li>
+            <li>Camping is the ultimate festival experience. You can camp for free or if you want a bit more comfort book your spot in our upgrade campsites with accommodations!</li>
+        </ul>
+        <ul class="include-list">
+            <li><i class="fa-solid fa-utensils"></i></li>
+            <li class="offer">Food & drinks</li>
+            <li>We will have over 400 food traders offering an array of exciting eats for Festival-goers. The choices are endless with a variety of delicious, world cuisine available across the site.</li>
+        </ul>
+        <ul class="include-list">
+            <li><i class="fa-solid fa-couch"></i></li>
+            <li class="offer">Chill zone</li>
+            <li>The area is meant as a quality chill-out zone. There will be cool daytime concerts and dj-sets during the afternoon and night, along with a bar and smoothie corner.</li>
+        </ul>
+        <ul class="include-list">
+            <li><i class="fa-solid fa-arrow-right-arrow-left"></i></li>
+            <li class="offer">Traders</li>
+            <li>Thousands of food traders, charities, campaigning organisations and sellers of the bizarre and unique apply to trade at our site each year</li>
+        </ul>
+        </div>
+        <div class="purchase-ticket">
+            <button type="button" class="purchase-button">Purchase ticket</button>
+        </div>
+        <div class="whole-program">
+            <a href="" class="program-link">See the whole program</a>
+        </div>
+    </section>
+    <section id="featured-bands">
```

Comment:
- [OPTIONAL] The heading in this section `Featured Artists` is not centered properly in this container. kindly consider centering it by putting it here and centering it with text-align: center; and then you can take another container for all the cards and apply display: grid; on that container instead of applying it in the section tag.
![qw](https://user-images.githubusercontent.com/96683843/234946472-130cdf36-e124-4927-9b82-711f931b0d32.jpg)